lms/envs/common.py

```diff
@@ -408,6 +408,9 @@
 
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
+
+    # Enable automatic status emails for ICRV attempts
+    'ENABLE_ICRV_STATUS_EMAILS': False,
```

Comment:
This is fine for now, but it should really be a Waffle switch. We shouldn't have to update configuration and redeploy to toggle this kind of feature. @clintonb do we have plans to introduce Waffle to edx-platform?